apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,331 @@
+
+| Property   | Description                                      |
+| ---------- | ------------------------------------------------ |
+| **deploy** | Defines if the contract is going to be deployed. |
+
+:::important
+For testing we suggest using the **FreeForAllGatekeeper**.
+:::
+
+#### MACI
+
+| Property           | Description                                 |
+| ------------------ | ------------------------------------------- |
+| **stateTreeDepth** | Defines how many users the system supports. |
+| **gatekeeper**     | Defines which gatekeeper will use.          |
+
+#### VkRegistry
+
+Verifying key registry used for generating proofs.
+
+| Property                | Description                                                                    |
+| ----------------------- | ------------------------------------------------------------------------------ |
+| **stateTreeDepth**      | Defines how many users the system supports.                                    |
+| **intStateTreeDepth**   | Defines how many ballots can be processed per batch when tallying the results. |
+| **messageTreeDepth**    | Defines how many messages (votes) the system supports.                         |
+| **voteOptionTreeDepth** | Defines how many vote options the system supports.                             |
+| **messageBatchDepth**   | Defines how many messages in a batch can the circuit process.                  |
+| **zkeys**               | Defines the path to the zkey files for QV and Non QV keys.                     |
+
+:::important
+The recommended values for test keys are: **10-1-2-2-1**. For ceremony keys: **14-5-9-3-2**.
+:::
+
+#### Poll
+
+| Property               | Description                                          |
+| ---------------------- | ---------------------------------------------------- |
+| **pollDuration**       | Defines how long is going to be the poll in seconds. |
+| **coordinatorPubkey**  | Defines the coordinator public MACI key.             |
+| **useQuadraticVoting** | Defines if the poll uses quadratic voting or not.    |
+
+### Deploy MACI Contracts
+
+To deploy the MACI contracts to a specific network you can append `:network` to the deployment commands, e.g. `pnpm deploy:sepolia` - please refer to the available networks on the package.json scripts section.
+
+```bash
+pnpm deploy:NETWORK
+```
+
+There are already some deployed contracts that could be reused, copy the `default-deployed-contracts.json` file if you need them to avoid deploying redundant contracts and save your gas fee.
+
+```bash
+cp default-deployed-contracts.json deployed-contracts.json
+```
+
+Delete any contract you want to redeploy and then run the following command to save gas:
+
+```bash
+pnpm deploy:NETWORK --incremental
+```
+
+## Deploy Poll
+
+To deploy your first Poll you can run the following command:
+
+```sh
+pnpm deploy-poll:NETWORK
+```
+
+:::important
```

Comment:
should we mention the `pnpm init-poll:NETWORK --poll [pollId]` here? either to put in the **notice** part to remind them if they don't init it, the poll never starts, or open a section called `## Start a poll` to explain this part?

Reply:
This is for MACI-Platform, right? the tutorial focuses on MACI

Reply:
you're right, sorry I get confused between repo :P